lib/backend/k8s/conversion/conversion_test.go

```diff
@@ -515,7 +515,7 @@ var _ = Describe("Test NetworkPolicy conversion", func() {
 							extensions.NetworkPolicyPeer{
 								PodSelector: &metav1.LabelSelector{
 									MatchLabels: map[string]string{
-										"k2": "v2",
+										"k2": "v3",
```

Comment:
same question here ..

Reply:
yeah I think you saw my comment.  This change wasn't necessary but it doesn't actually make any difference - we're just testing a different label value.

Reply:
i'll change.  it's beginning to bug me now.